crates/autopilot/src/run.rs

```diff
@@ -400,8 +401,10 @@ pub async fn run(args: Arguments) {
         .unwrap();
     let native_price_estimator = price_estimator_factory
         .native_price_estimator(
+            args.native_price_estimators.as_slice(),
             &PriceEstimatorSource::for_args(
-                args.native_price_estimators.as_slice(),
+                //builtin estimators are passed in separately
+                &[],
```

Comment:
This feels like a code smell.

Reply:
Maybe `GenericPriceEstimator` shouldn't be conflated with the "special native price estimators" (i.e. the 1Inch price API estimator).

Reply:
I agree that this is a smell, but how would your suggestion work? Then we would need to specify
1. BuiltIn Native Only Price estimator
2. Builtin Price Estimator (would all general estimators also become price estimators?)
3. LegacyPriceEstimators
4. External (co-located) price estimators

Would 1 and 2 go into the same stage or in separate stages?

What I'd like to see eventually is to just specify two things:
1. A list of external price estimator
2. A list of list of native estimators (if a value is not in 1, it needs to be instantiated in place)

I'm just not entirely sure how to best get there.

Reply:
I also see it as:
1. A list of external price estimator (so `--price-estimators + --price-estimation-drivers + --legacy-price-estimators`)
2. A list of list of native estimators (1Inch).

In terms of stages, wouldn't this always need to be configurable in some way? Specifically, how does the racing native price estimator know that the `Baseline` `--price-estimation-drivers` can go in the first stage (i.e. cheap and quick) but not the `0x` `--price-estimation-drivers`?

Reply:
In the next PR it is for now configured such that the builtin estimators go first and then we query the external estimators in a second stage.